docs/backup.md

```diff
@@ -0,0 +1,36 @@
+`symbol-bootstrap backup`
+=========================
+
+The command backups the Mongo and RocksDb data folder into a Zip file that can be used for --backupSync feature. Bootstrap compose services must be stopped before calling this command.
+
+Note: this command is designed for NGL to be used when running public main or public test networks. It's not backing up any node specific information.
```

Comment:
Is this true? Couldn't it be used by a private network to speed up sync?

Reply:
Yes, it can be used. Atm we don't have a well-documented process of a user joining a private network with bootstrap. Pieces are there, I just need to put them together. Main issue is how to easily share a private network preset and seed in new private nodes

Reply:
It's just a warning for the current users that may want to use it on testnet backups. This backup is not backing a node (like's node configuration or keys), it's just the data that can be shared between multiples nodes